### ISSUE TITLE
3D VAE finetune

### ISSUE THREAD

**Issue by Simona0212** [THUDM/CogVideo#111]
我们的科研项目想要在3D VAE的基础上进行改动和微调。我们尝试在原有模块的训练代码上进行修改，发现有点困难。
请问能否开源3D VAE模块的训练代码和配置文件？
感谢CogVideoX团队！

**Comment by Simona0212** [comment]
I am going to modify and finetune the 3D VAE for our research project. We have tried to modify the training code of the original module and found it a little difficult. I would highly appreciate if you could provide the code and configuration  files required to train and finetune the 3D VAE module (for video encoding and video decoding). Thank you very much!

**Comment by zRzRzRzRzRzRzR** [comment]
Fine-tuning the VAE alone doesn’t seem very meaningful. If your goal is to fine-tune in conjunction with Transformers while reducing memory usage in the VAE encoder, we will continue working on this optimization. You can check for updates here.
#194 https://github.com/huggingface/diffusers/pull/9302

**Comment by vanche** [comment]
@serend1p1ty I'm really interested in the tricks you've used and how much you've improved the VAE's performance. Could you share more details about the techniques you applied and the results you achieved?

**Comment by serend1p1ty** [comment]
@vanche Sorry I cannot share more tech detail. Here is my results.
![image](https://github.com/user-attachments/assets/34497fc5-f516-445b-a2fe-33e203576232)
There is still a lot of room for improvement (loss has not converged).

**Comment by vanche** [comment]
@serend1p1ty Thank you for sharing the performance results. Do you plan to publish a paper or technical report?

**Comment by serend1p1ty** [comment]
@vanche Basically, it's just some increment improvements, and unable to support a paper.

**Comment by derekcheng08** [comment]
@serend1p1ty - very interesting, thanks for sharing. Did you then need to fine-tune CogVideoX to work with your VAE?
